stubs/defaults.stub

```diff
@@ -0,0 +1,258 @@
+                    { key: 'wysiwyg', ignore: true },
+                ],
+            },
+            {
+                key: 'data',
+                ignore: false,
+                components: [
+                    { key: 'inputFormat', defaultValue: 'html', disabled: true },
+                ],
+            }
+        ],
+        email: [
+            {
+                key: 'data',
+                ignore: false,
+                components: [
+                    { key: 'inputFormat', ignore: true },
+                ],
+            },
+            {
+                key: 'validation',
+                ignore: false,
+                components: [
+                    { key: 'kickbox', ignore: true },
+                ],
+            },
+        ],
+        address: [
+            {
+                key: 'provider',
+                ignore: false,
+                components: [
+                    { key: 'provider', defaultValue: 'nominatim', disabled: true },
+                    { key: 'manualModeViewString', ignore: true},
+                ],
+            }
+        ],
+        datetime: [
+            {
+                key: 'date',
+                ignore: false,
+                components: [
+                    { key: 'datePicker.disable', ignore: true }, // feature is broken, so it has been disabled
+                ],
+            },
+            {
+                key: 'data',
+                ignore: false,
+                components: [
+                    { key: 'customOptions', ignore: true },
+                    { key: 'defaultDate', ignore: true }, // depends on JS eval. it can work (this JS is client-side-only), but it seems dangerous.
+                ]
+            }
+        ],
+        select: [
+            {
+                key: 'data',
+                ignore: false,
+                components: [
+                    { key: 'dataSrc', defaultValue: 'values', disabled: true },
+                    { key: 'idPath', ignore: true },
+                    { key: 'template', ignore: true },
+                    { key: 'refreshOn', ignore: true },
+                    { key: 'refreshOnBlur', ignore: true },
+                    { key: 'clearOnRefresh', ignore: true },
+                    { key: 'customOptions', ignore: true },
+                ],
+            }
+        ],
+        time: [
+            {
+                key: 'data',
+                ignore: false,
+                components: [
+                    { key: 'dataFormat', ignore: true },
+                ],
+            }
+        ],
+        file: [
+            {
+                key: 'file',
+                ignore: false,
+                components: [
+                    { key: 'storage', defaultValue: 'url', disabled: true },
+                    { key: 'url', defaultValue: '/dynamic-forms/storage/url', disabled: true },
```

Comment:
This is a big file to duplicate for this relatively small adjustment.

Let me suggest an alternative: [Laravel Mix environment variables](https://laravel.com/docs/8.x/mix#environment-variables). If you add a `MIX_FORMIO_UPLOAD_TYPE=something` to the `.env` during installation, you can check `process.env.MIX_FORMIO_UPLOAD_TYPE` in JS to adjust this section accordingly.

Reply:
I agree, will try and implement, was looking at some way of manipulating this file from the installer but I couldn't find it.